samples/star/src/main/kotlin/com/slack/circuit/star/petlist/PetList.kt

```diff
@@ -390,8 +392,8 @@ private fun PetListGridItem(animal: PetListAnimal, onClick: () -> Unit = {}) {
         modifier = Modifier.fillMaxWidth().aspectRatio(1f).testTag(IMAGE_TAG),
         model =
           ImageRequest.Builder(LocalContext.current)
-            .data(animal.imageUrl)
-            .memoryCacheKey(animal.imageUrl)
+            .data(updatedImageUrl)
+            .memoryCacheKey(memCacheKey)
```

Comment:
Just don't call memoryCacheKey if it's null I think?

Reply:
`memoryCacheKey` actually takes in a nullable`String?` and computes a memory cache key if it's null. So I just put `animal.imageUrl` back here, since it knows what to do if it's null.